### ISSUE TITLE
A11y_Azure Tools Storage Explorer_Disk_Paste Disk_NonTextContrast : The color contrast ratio of the control border with respect to background is less than 3:1 when focus goes on the "Paste" button .

### ISSUE THREAD

**Issue by shreyashi227** [microsoft/AzureStorageExplorer#3459]
**GitHubTags**: #C&AI_Aug2020_BMAzure Tools Storage Explorer;#A11yWCAG2.1;#A11YTCS;#DesktopApp;#Azure Tools Storage Explorer;#WCAG1.4.11;#Win32;#A11ySev2;#Benchmark;#ColorContrast;#AI4D;#FTP;

### **Environment Details:**
Application Name: Azure Tool Storage Explorer
Application version: 1.14.2
Windows Version - Windows10

### **Repro Steps:**

1. Launch Storage Explorer
2. Navigate to "Open Connect dialog" pane and Sign in to Azure.
3. Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Apply" button.
4. Navigate to "Disks" of the selected subscription.
5. Navigate to "A11ytest" and  click on it.
6. Navigate to Paste Disk button 
7. Open AI4D and Check if color contrast ratio of the control border with respect to background is less than 3:1 when focus goes on the "Paste" button . 

### **Actual:**
The color contrast ratio of the control border w.r.t background when focus goes on the "Paste" button is 1.492:1 which is less than 3:1 

### **Similar issue is observed for below scenario:**
**1. Disk - Delete Disk**

1. Launch Storage Explorer
2. Navigate to "Open Connect dialog" pane and Sign in to Azure.
3. Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Apply" button.
4. Navigate to "Disks" of the selected subscription.
5. Navigate to "A11ytest" and  click on it.
6. Navigate to Delete Disk button 
7. Check if color contrast ratio of the control border with respect to background is less than 3:1 when focus goes on the "Delete" button . 

### **Expected:**
The color contrast ratio of the control border w.r.t background when focus goes on the "Paste" button should be equal or greater than 3:1 

### **User Impact:**
Low vision users may not able to see the control has keyboard focus.

### **Attachment**
[https://microsoftapc-my.sharepoint.com/:i:/g/personal/v-shrkh_microsoft_com/EeI7C3MxXgxPoD90pPRA5qgB3rspdvuFjtmdysPGOy23oQ?e=9eAeD](https://microsoftapc-my.sharepoint.com/:i:/g/personal/v-shrkh_microsoft_com/EeI7C3MxXgxPoD90pPRA5qgB3rspdvuFjtmdysPGOy23oQ?e=9eAeDR)

**Comment by MRayermannMSFT** [comment]
![image](https://user-images.githubusercontent.com/21112697/96174668-7dbd0400-0ede-11eb-8069-bf60539f891d.png)

**Comment by shreyashi227** [comment]
Verified the issue and the issue is fixed as color contrast ratio of focus border is greater than 3:1. So closing this bug.

![image](https://user-images.githubusercontent.com/60881447/96447389-73c53900-122f-11eb-8fce-90744cbd651c.png)

**Comment by fsteffi** [comment]
**GitHub Tags:**#A11yMAS;

**Comment by 12-shweta** [comment]
Github Tags:#Closed;
